features/lesson/src/main/kotlin/com/teacherworkout/features/lesson/LessonFeature.kt

```diff
@@ -0,0 +1,42 @@
+package com.teacherworkout.features.lesson
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.*
+import androidx.navigation.compose.composable
+import com.teacherworkout.features.lesson.di.lessonModule
+import com.teacherworkout.commons.ui.navigation.AppDestinations
+import org.koin.androidx.compose.getViewModel
+import org.koin.core.context.GlobalContext.loadKoinModules
+import org.koin.core.parameter.parametersOf
+
+//TODO: add a navigation graph instead of a simple `composable`
```

Comment:
It might be a better idea to create issues for all this instead of adding comments? What do you think about this?

Reply:
I didn't give a thought to that, but now I think that it could be turned into an issue. But even so, I can't do that before the pull request is merged, right? Sorry if I'm being ignorant :disappointed_relieved:

Reply:
Right, normally we would create an issue after the PR is merged.